x-pack/plugins/maps/public/routing/state_syncing/global_sync.ts

```diff
@@ -3,27 +3,24 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-
+import { QueryState } from 'src/plugins/data/public';
 import { syncQueryStateWithUrl } from '../../../../../../src/plugins/data/public';
 import { getData } from '../../kibana_services';
-// @ts-ignore
 import { kbnUrlStateStorage } from '../maps_router';
 
 export function startGlobalStateSyncing() {
   const { stop } = syncQueryStateWithUrl(getData().query, kbnUrlStateStorage);
   return stop;
 }
 
-export function getGlobalState() {
-  return kbnUrlStateStorage.get('_g');
+export function getGlobalState(): QueryState {
```

Comment:
`QueryState` is not quite the right type for Maps global state. Maps global state lacks `query`. Mind creating a defined type in the this file that looks like

```
export interface MapsGlobalState {
  time?: TimeRange;
  refreshInterval?: RefreshInterval;
  filters?: Filter[];
}
```

Reply:
Replaced QueryState with MapsGlobalState.